src/readbin/ground_truth.ml

```diff
@@ -0,0 +1,26 @@
+open Bap.Std
+open Core_kernel.Std
+open Or_error
+
+
+let from_unstripped_bin bin : Addr.Set.t t =
+  let tmp = Filename.temp_file "bw_" ".symbol" in
+  let cmd = Printf.sprintf "bap-byteweight dump -i %s %S > %S" "symbols"
+      bin tmp in
+  if Sys.command cmd = 0
+  then return (In_channel.with_file tmp ~f:Symbols.read_addrset)
+  else error_string cmd
```

Comment:
This error message wouldn't be very informative, use `errorf` function to build nice error messages, e.g.,
```
errorf "failed to fetch symbols from unstripped binary, command `%s' failed" cmd
```